decentraland/unity-renderer
# feat: add view all option in passport collectibles tab

## What does this PR change?
Fix #4028
Adds VIEW ALL button to see all NFTs (available for WEARABLES, NAMES and LANDS) in the passport.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?explorer-branch=feat/view-all-collectibles&DISABLE_force_to_request_wearables_through_kernel **<--- Make sure you're using the `DISABLE_force_to_request_wearables_through_kernel` url param**
2. Go to the passport of anyone.
3. Go to the collectibles tab.
4. Check that the "View all" button appears when some of the carousels has more than 40 items loaded (this doesn't apply for EMOTES carousel).
5. Click in the "View all" button and check that all the items are loaded in a paginated way.
6. Navigate through the different pages and check that everything works fine.
7. Hover the mouse on each card and check that the corresponding visual effect works.
8. Click on the BUY button of each card (this only applies to WEARABLES) and check that you're being redirectioned to the correct marketplace url.
9. Do the same test for several users (even for your own passport).

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
